zkevm-circuits/src/witness/rw.rs

```diff
@@ -1071,3 +1071,28 @@ impl From<&operation::OperationContainer> for RwMap {
         Self(rws)
     }
 }
+
+/// RwTablePermutationFingerprints
+#[derive(Debug, Default, Clone)]
+pub struct RwTablePermutationFingerprints<F> {
```

Comment:
In #1690, I actually make a `Chunk<F>` that holds all chunk-related data, so l guess if this struct is incorporated in `Chunk<F>` then it wouldn't be necessary anymore? 
For now it makes total sense so just keep it, just a heads up I'll probably get rid of it when I merge :)